projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/cds/profile-tag.ts

```diff
@@ -0,0 +1,135 @@
+export const ProfileTagHelper = {
+  assertPageViewEvent(xhr: Cypress.WaitXHR) {
+    ProfileTagHelper.assertConsentReference(xhr);
+    expect(xhr.requestBody['_profile_custom'].schema).to.equal(
+      'context/core/pageViewed'
+    );
+  },
+  assertProductViewEvent(xhr: Cypress.WaitXHR) {
+    ProfileTagHelper.assertConsentReference(xhr);
+    expect(xhr.requestBody['_profile_custom'].schema).to.equal(
+      'context/commerce/ProductView'
+    );
+  },
+  assertConsentReference(xhr: Cypress.WaitXHR) {
+    expect(xhr.requestHeaders['consent-reference']).to.equal(
+      ProfileTagHelper.consentReferenceResponse.id
+    );
+  },
+  consentReferenceResponse: {
+    id: 'faa6e4b3-887c-4c99-94a9-7b14414df604',
+    link:
+      'http://htv730345051.api.eu.context.cloud.sap/htv730345051/consents/faa6e4b3-887c-4c99-94a9-7b14414df604',
+    consentReferenceToken:
+      'aepsqnr50TJfVEzegIibGDLiskrJEuiG0jTj6qaWxNSXgk1GZmIz5KsfqVJpqZP3Tg3PZAG5GGHW6DiN_8L-rA',
+  },
+  configResponse: {
```

Comment:
maybe put this into a variable

Reply:
It is currently in the variable ProfileTagHelper.configResponse. Did you mean that you would rather me create a class with static variables here over creating a javascript object literal (typescript will give the same intellisense either way AFAIK)

Reply:
it's fine